webhooks/php/webhook_callback.php

```diff
@@ -1,10 +1,36 @@
 <?php
-// Read more on https://talkjs.com/docs/Webhooks/Getting_Started.html
-// Parse the request body as JSON
+// This is a vanilla PHP example to read a webhook and validate its integrity. 
+// Some lines are different if you use Laravel, those are shown in comments.
 
-$input = file_get_contents("php://input");
-$event = json_decode($input);
 
+// Read the POST body in full + relevant headers
+$payload = file_get_contents("php://input");
+$signature = $_SERVER["HTTP_X_TALKJS_SIGNATURE"];
+$timestamp = $_SERVER["HTTP_X_TALKJS_TIMESTAMP"];
+// Laravel:
+//   $payload = $request->getContent();
+//   $signature = $request->header("X-TalkJS-Signature");
+//   $timestamp = $request->header("X-TalkJS-Timestamp");
+
+
+// Your secret key, as found in the TalkJS dashboard. Make sure that you never
+// share your secret key anywhere nor commit it to code repositories.
+$secret = "YOUR_SECRET_KEY";
+// Laravel:
+//   $secret = env("TALKJS_SECRET_KEY");
```

Comment:
Maybe make the laravel and PHP examples behave the same here, and change the vanilla example to read from an environment variable as well? Now the way the two examples behave differently feels weird to me. Then again, maybe pulling environment variables into this adds too much complexity for someone who just wants to plunk this file on a basic apache server and be done with it.